commands/namespace.go

```diff
@@ -86,25 +86,28 @@ var namespaceGetCmd = &cobra.Command{
             }
         }
 
-        namespace, _, err := Client.Namespaces.Get(qualifiedName.GetNamespace())
-
-        if err != nil {
-            whisk.Debug(whisk.DbgError, "Client.Namespaces.Get(%s) error: %s\n", getClientNamespace(), err)
+        errHandler := func(err error, kind string) error {
```

Comment:
for consistency with other existing error handler defintions, i suggest moving this error handler outside of the `RunE` function definition... in this same module though.